docs/ibc/overview.md

```diff
@@ -174,7 +174,7 @@ Once an acknowledgement is received successfully on the original sender the chai
 If you want to learn more about IBC, check the following specifications:
 
 * [IBC specification overview](https://github.com/cosmos/ics/blob/master/ibc/README.md)
-* [IBC SDK specification](../../modules/ibc)
+* [IBC SDK specification](https://docs.cosmos.network/v0.41/modules/ibc/)
```

Comment:
```suggestion
* [IBC SDK specification](../../x/ibc/spec/README.md)
```

would prefer to avoid absolute links. This should work.

Reply:
makes sense to me!